src/manifold.cpp

```diff
@@ -71,8 +71,12 @@ MeshGLP<Precision, I> GetMeshGLImpl(const manifold::Manifold::Impl& impl,
 
   MeshGLP<Precision, I> out;
   out.numProp = 3 + numProp;
-  out.tolerance = std::max(std::numeric_limits<Precision>::epsilon(),
-                           static_cast<Precision>(impl.tolerance_));
+  out.tolerance = impl.tolerance_;
+  if (std::is_same<Precision, float>::value)
+    out.tolerance =
+        std::max(out.tolerance,
+                 static_cast<Precision>(std::numeric_limits<float>::epsilon() *
+                                        impl.bBox_.Scale()));
```

Comment:
Should we perform simplification after this?

Reply:
Huh, that's an interesting idea. It feels kind of wrong because you'd have to pay the cost of an extra simplification check on every mesh output, as well as paying the cost of extra faces in all of the boolean operations up to that point. It would be much better to set tolerance to the right level earlier instead. I feel like we should design our API to push people toward that golden path.